packages/ui-extensions/buildTargetDts.ts

```diff
@@ -81,6 +81,24 @@ export type GlobalThis = typeof globalThis & {
   shopify: Api;
 }\n`;
 
+  if (surface === 'customer-account') {
+    template = `import type {ExtensionTargets} from '../extension-targets';
+import {ExtensionNavigation} from '../api';
+type Target = ExtensionTargets[${name}];
+export type Api = Target['api'];
+export type Output = Target['output'];
+export type Navigation = ExtensionNavigation<${name}>;
+
+export type GlobalThis = typeof globalThis & {
+  shopify: Api;
+}\n
+
+declare global {
```

Comment:
Instead of re-declaring can you try adding this at the top?
```
import('../globals');
```

Reply:
So I noticed that in globals.ts I dont have the target to be able to do something like `ExtensionNavigation<${name}>`. 

ExtensionNavigation is different based on the target.